xdsl/irdl.py

```diff
@@ -29,6 +30,7 @@ class VerifyException(DiagnosticException):
 
 class IRDLAnnotations(Enum):
     ParamDefAnnot = 1
```

Comment:
Would you mind adding a docstring for those vars?

Reply:
Good idea 👍